addons/copy-reporter/userscript.js

```diff
@@ -0,0 +1,81 @@
+export default async function ({ addon, console, msg }) {
+  addon.tab.createEditorContextMenu(
+    (ctx) => {
+      // ugly hack to get the string value
+      let element = ctx.target.lastChild.lastChild;
+
+      if (ctx.type === "monitor_slider") {
+        element = element.children[element.children.length - 2].lastChild;
+      } else {
+        element = element.lastChild;
+        if (ctx.type === "monitor_default") {
+          element = element.lastChild;
+        }
+      }
+
+      if (element.innerText.length !== 0) {
+        navigator.clipboard.writeText(element.innerText);
+      }
+    },
+    {
+      className: "copy",
+      types: ["monitor_default", "monitor_large", "monitor_slider"],
+      position: "monitor",
+      order: 0,
+      label: msg("copy-value"),
+    }
+  );
+
+  // add button to reporter bubble
+  const ScratchBlocks = await addon.tab.traps.getBlockly();
+
+  // https://github.com/LLK/scratch-blocks/blob/893c7e7ad5bfb416eaed75d9a1c93bdce84e36ab/core/workspace_svg.js#L979
+  ScratchBlocks.WorkspaceSvg.prototype.reportValue = function (id, value) {
+    let block = this.getBlockById(id);
+    if (!block) {
+      throw "Tried to report value on block that does not exist.";
+    }
+
+    ScratchBlocks.DropDownDiv.hideWithoutAnimation();
+    ScratchBlocks.DropDownDiv.clearContent();
+
+    let contentDiv = ScratchBlocks.DropDownDiv.getContentDiv();
+
+    let valueReportBox = document.createElement("div");
+    valueReportBox.setAttribute("class", "valueReportBox");
+    valueReportBox.innerHTML = ScratchBlocks.scratchBlocksUtils.encodeEntities(value);
+    valueReportBox.style.userSelect = "text";
+    // use to get focus and event priority
+    valueReportBox.setAttribute("tabindex", "0");
+    // if the user pressed Ctrl+C, prevent propagation to Blockly
+    valueReportBox.onkeydown = (event) => {
+      if ((event.altKey || event.ctrlKey || event.metaKey) && event.keyCode == 67) {
+        event.stopPropagation();
+      }
+    };
+
+    if (value.length !== 0) {
+      const copyButton = document.createElement("img");
+      copyButton.setAttribute("role", "button");
+      copyButton.setAttribute("tabindex", "0");
+      copyButton.setAttribute("alt", msg("copy-to-clipboard"));
+      // cannot be done in CSS because of a bug
```

Comment:
Could the SVG itself be set to that size instead then?

Reply:
That would invole changing that actual path.

Reply:
No, I mean opening the SVG in an editor and scaling jt to 14x14. It shouldn't effect tge quality because it's a vector.

Reply:
It's usually a good idea to set width and height explicitly, using either CSS or `width` and `height` attributes.

What's the bug that makes it impossible to do this using CSS here?

Reply:
I tried setting the width and height with CSS just now and didn't have any issues. It was probably a caching issue.

Reply:
I'm using Firefox so it's possible something to do with that.

Reply:
Maybe it was a Firefox problem?

Reply:
Were the styles just not getting applied?

Reply:
The attribute was populated :shrug:.

Reply:
I'm not sure what could cause a bug like that. The following worked for me on both Edge and Firefox:
```js
copyButton.style.width = "14px";
copyButton.style.height = "14px";
```

Adding these to the userstyle also worked:
```css
width: 14px;
height: 14px;
```

Reply:
Try setting the styles again and shift-click the reload page button to ignore the browser cache.

Reply:
I don't think the cache applies to attributes though. I'll still try.

Reply:
Hmm... it does seem to work, but it's not centred any more. I think something else caused that though.